Lib/ldif.py

```diff
@@ -633,7 +633,7 @@ def handle(self,dn,entry):
 def ParseLDIF(f,ignore_attrs=None,maxentries=0):
   """
   Parse LDIF records read from file.
-  This is a compatibility function. Use is deprecated!
+  This is a compatibility function.
```

Comment:
``warnings.warn('ParseLDIF is deprecated, use LDIFRecordList.parse() instead.', category=DeprecationWarning, stacklevel=2)``